src/hooks/plan_tree_walker.c

```diff
@@ -4,9 +4,19 @@
 
 #include <miscadmin.h>
 #include <nodes/nodeFuncs.h>
+#include <nodes/nodes.h>
 #include <nodes/parsenodes.h>
 #include <nodes/plannodes.h>
 
+bool is_plan_tree_node(Node *node)
```

Comment:
I think if you make this `static inline` and put it in the header file (`src/hooks/plan_tree_walker.h`) it will be more likely to be inlined by the complier